src/test/java/org/folio/rest/api/FixedDueDateApiTest.java

```diff
@@ -91,7 +92,7 @@ public void canCreateSchedule()
       ResponseHandler.empty(updateBad3Completed));
     Response updateBad3Response = updateBad3Completed.get(5, TimeUnit.SECONDS);
     assertThat(String.format("Failed to create due date: %s", representation.encodePrettily()),
-      updateBad3Response.getStatusCode(), is(422));
+      updateBad3Response.getStatusCode(), is(HttpStatus.HTTP_UNPROCESSABLE_ENTITY.toInt()));
```

Comment:
I might be tempted to statically import these constants to improve the readability of these assertions